serialization.md

```diff
@@ -422,14 +362,13 @@ bin/my-app-binary
 bin/my-app-tools
 ```
 
-The TarSum checksum for the archive file is then computed and placed in the
-JSON metadata along with the execution parameters.
+The TarSum checksum for the archive file is then computed and placed in the JSON metadata along with the execution parameters.
```

Comment:
ah. nice.

On Mon, Apr 11, 2016 at 3:19 PM, Stephen Day <notifications@github.com>
wrote:

> In serialization.md
> <https://github.com/opencontainers/image-spec/pull/12#discussion_r59264196>
> :
>
> > @@ -422,14 +362,13 @@ bin/my-app-binary
> >  bin/my-app-tools
> >  ```
> >
> > -The TarSum checksum for the archive file is then computed and placed in the
> > -JSON metadata along with the execution parameters.
> > +The TarSum checksum for the archive file is then computed and placed in the JSON metadata along with the execution parameters.
>
> @vbatts <https://github.com/vbatts> That work item is docker/docker#21938
> <https://github.com/docker/docker/issues/21938>, for reference.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/12/files/c146488d55d702f2c7103da26ac5a20dfc5f34c1#r59264196>
>